libs/blocks/media/media.js

```diff
@@ -30,14 +30,15 @@ function decorateAvatar(el) {
 function decorateQr(el) {
   const text = el.querySelector('.text');
   if (!text) return;
-  const appStore = text.children[(text.children.length - 1)];
-  const googlePlay = text.children[(text.children.length - 2)];
+  const appStore = text.children[(text.children.length - 1)]?.querySelector('a');
+  const googlePlay = text.children[(text.children.length - 2)]?.querySelector('a');
   const qrImage = text.children[(text.children.length - 3)];
+  if (!qrImage || !appStore || !googlePlay) return;
```

Comment:
Could we have situations where we just have a QR code or just store apps or just one store app? Judging by the rigidity of the markup, I'd assume not, but this feels like it might come back to haunt us. I'd vote for `qrImage?.classList.add('qr-code-img');` and so on for the next lines, but maybe I'm overly cautious.

Reply:
I was wondering about the same thing! But the current code would throw an error and optional chaining assignments are currently disabled by our linter and according to MDN https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining#invalid_optional_chaining

Reply:
We can also do this, if you prefer:

```js
  if(appStore) {
    appStore.classList.add('app-store');
    appStore.textContent = '';
  }
  if(googlePlay) {
    googlePlay.classList.add('google-play');
    googlePlay.textContent = '';
  }
  qrImage?.classList.add('qr-code-img');
```

Reply:
According to the requirements in the ticket, the QR code is supposed to turn into the app store links, so I would assume we should always have all 3